app/views/user_mailer/in_person_failed.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="s16">
+  <%= t('user_mailer.in_person_verified.greeting') %><br>
+  <%= t(
+        'user_mailer.in_person_failed.intro',
+        location: @presenter.location_name,
+        date: @presenter.formatted_verified_date,
+      ) %><br /><br />
+  <%= t('user_mailer.in_person_failed.body') %><br><br />
+  <strong><%= t('user_mailer.in_person_failed.verifying_identity') %></strong><br>
```

Comment:
Should these be paragraphs?

```suggestion
  <p>
    <%= t(
          'user_mailer.in_person_failed.intro',
          location: @presenter.location_name,
          date: @presenter.formatted_verified_date,
        ) %>
  </p>
  <p>
    <%= t('user_mailer.in_person_failed.body') %>
  </p>
  <p><strong><%= t('user_mailer.in_person_failed.verifying_identity') %></strong></p>
```

Reply:
made into paragraphs in e87a1979eeb6f903b2d66225e73508dc6b6c5654